app/views/helpers/claim-expense-helper.js

```diff
@@ -26,6 +27,9 @@ module.exports = function (expense) {
     case 'ferry':
       formattedDetail = `${addTicketOwnerPrefix(expense)}${expense.From} to ${expense.To} as ${ExpenseTypeEnum.FERRY.ticketType[expense.TicketType]}${addReturnPostfix(expense)}`
       break
+    case 'car':
+      formattedDetail = `${expense.From} to ${displayHelper.getPrisonDisplayName(expense.To)}`
+      break
```

Comment:
Tested adding a toll charge and parking charge, they still show the prison value instead of display name so you'll need to make this same change to apply to those too